worker/upload.go

```diff
@@ -206,7 +216,7 @@ func (w *worker) threadedUploadPackedSlabs(rs api.RedundancySettings, contractSe
 
 	// keep uploading packed slabs until we're done
 	ctx := context.WithValue(w.shutdownCtx, keyInteractionRecorder, w)
-	for {
+	for !w.isStopped() {
 		uploaded, err := w.uploadPackedSlabs(ctx, defaultPackedSlabsLockDuration, rs, contractSet, lockPriority)
```

Comment:
This is still being interrupted immediately when `w.shutdownCtx` is closed.

Reply:
I did that on purpose, I don't think we need to upload packed slabs during a graceful shutdown. Or at least that was my decision, could've gone either way. Don't you feel this needlessly overcomplicates things? If you have upload packing enabled I'm sure you don't really care whether some packed slabs get uploaded when you shut the renter down. 

The main driver was that if you have several packed slabs in the queue, uploading them might "drain" the main shutdown context causing a) the worker shutdown to time out and b) leaving little room (the reserved part of the context) for the other services to shut down.

Reply:
I don't mean for us to upload all of them. Only to not abort a single slab upload mid-upload. e.g. we might be done uploading the slab to the host already and just need to notify the backend but don't do that because we interrupt. After the upload it's fine to interrupt before starting a new one.